mne/report.py

```diff
@@ -949,46 +949,51 @@ def _validate_input(self, items, captions, section, comments=None):
 
         return items, captions, comments
 
-    def _add_figs_to_section(self, figs, captions, section='custom',
```

Comment:
This private function was used exactly once, so I refactored it out. I think it was meant to be re-used at some point, but never was in the end.